extensions/htlc/besu-eth/src/main/typescript/generated/openapi/typescript-axios/.gitignore

```diff
@@ -0,0 +1,4 @@
+wwwroot/*.js
```

Comment:
@AzaharaC Apologies for not mentioning this one earlier: this file should also be removed from version control and then it should not be re-generated by OpenAPI generator anymore.